project-base/storefront/components/Pages/Wishlist/Wishlist.tsx

```diff
@@ -31,59 +31,65 @@ export const Wishlist: FC = () => {
         );
     };
 
-    if (fetching) {
-        return <LoaderWithOverlay />;
-    }
-
     return (
         <>
-            <Heading className="!text-big lg:!text-h1" type="h1">
-                {t('Wishlist')}
+            <Heading type="h1">
+                {`${t('Wishlist')}${wishlist?.products.length ? ` (${wishlist.products.length})` : ''}`}
             </Heading>
-            {!!wishlist?.products.length && (
-                <div className="flex w-full flex-col items-center justify-between border-b border-greyLighter pb-2 lg:flex-row">
-                    <div
-                        className="mb-2 cursor-pointer items-center rounded bg-greyVeryLight py-2 px-4 transition-colors hover:bg-greyLighter sm:inline-flex lg:mb-0"
-                        onClick={() => {
-                            handleCleanWishlist();
-                        }}
-                    >
-                        <span className="mr-3 text-sm">{t('Delete all from wishlist')}</span>
-                        <RemoveThinIcon className="w-3" />
-                    </div>
-                    <div className="flex w-full flex-col items-center lg:w-1/2 lg:flex-row">
-                        <TextInput
-                            id="copyUrl-input"
-                            label={t('Send a link to this wishlist')}
-                            value={buildShareUrl()}
-                            onChange={() => {
-                                return false;
-                            }}
-                        />
-                        <SubmitButton
-                            className="mt-2 lg:ml-2 lg:mt-0"
+
+            {fetching && <SkeletonModuleWishlist />}
+
+            {wishlist?.products.length && !fetching && (
```

Comment:
This is a different condition than in comparison. Here you check the length, there you only check the products array. Is this on purpose?

Reply:
Refactored, according to previous comment.